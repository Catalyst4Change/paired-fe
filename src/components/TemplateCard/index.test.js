import React from 'react';
import { shallow } from 'enzyme';
import { TemplateCard } from './index';

describe('TemplateCard', () => {
  it('should match the snapshot when type is giving-help', () => {
    const wrapper = shallow(<TemplateCard type='giving-help' />);
    expect(wrapper).toMatchSnapshot();
  });
  
  it('should match the snapshot when type is receiving-help', () => {
    const wrapper = shallow(<TemplateCard type='receiving-help' />);
    expect(wrapper).toMatchSnapshot();
  });
  
  it('should match the snapshot when type is openings', () => {
    const wrapper = shallow(<TemplateCard type='openings' />);
    expect(wrapper).toMatchSnapshot();
  });
});