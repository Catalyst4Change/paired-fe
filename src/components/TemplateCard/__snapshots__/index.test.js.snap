// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemplateCard should match the snapshot when type is giving-help 1`] = `
<div
  className="ScheduleCard TemplateCard"
>
  <p>
    You haven't been booked yet. Check back soon. 
     
    <span
      aria-label="smiley emoji"
      role="img"
    >
      ðŸ™‚
    </span>
  </p>
</div>
`;

exports[`TemplateCard should match the snapshot when type is openings 1`] = `
<div
  className="ScheduleCard TemplateCard"
>
  <p>
    You have no openings. 
     
    <span
      aria-label="crying emoji"
      role="img"
    >
      ðŸ˜­
    </span>
  </p>
</div>
`;

exports[`TemplateCard should match the snapshot when type is receiving-help 1`] = `
<div
  className="ScheduleCard TemplateCard"
>
  <p>
    Sign up to pair. Coding is better with friends. 
     
    <span
      aria-label="100 emoji"
      role="img"
    >
      ðŸ’¯
    </span>
  </p>
</div>
`;
